playbooks/common/openshift-cluster/scaleup.yml

```diff
@@ -1,6 +0,0 @@
----
-- include: evaluate_groups.yml
-
-- include: ../openshift-node/config.yml
-  vars:
-    openshift_deployment_type: "{{ deployment_type }}"
```

Comment:
We'll need to make sure that the docs get updated for this change (or at least the docs PR/Issue and/or the docs bz)